.github/workflows/ci.yml

```diff
@@ -10,14 +10,15 @@ jobs:
       matrix:
         node: [
           14,
-          13,
```

Comment:
removed this since node 13 is no longer supported but I could put it back if it seems valuable to still test for it here